app/javascript/chat/message.jsx

```diff
@@ -60,6 +61,22 @@ const Message = ({
       </div>
     </div>
   );
+  const dropdownReport = (
+    <div className="message__actions">
+      <span className="ellipsis__menubutton">
+        <img src={ThreeDotsIcon} alt="dropdown menu icon" />
```

Comment:
I can't seem to set my local instance up to test this properly, but from looking at the markup it appears this isn't an accessible dropdown (`span` and `img` elements are not focusable or keyboard operable). How does this work from a keyboard/screen reader when rendered?